extensions/qute/runtime/src/main/java/io/quarkus/qute/i18n/Message.java

```diff
@@ -47,9 +53,10 @@
     String key() default DEFAULT_NAME;
 
     /**
+     * This value has higher priority over a message template specified in a localized file.
```

Comment:
I think that we should mention that 

1. It's a good practice to specify the value,
2. If no value is provided and no localized file matches then the build fails.

Reply:
sure, thanks